Sources/GRPC/MessageEncodingHeaderValidator.swift

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
This whole file was copied verbatim from the (now deleted) HTTP1ToGRPCServerCodec.swift